### DISCUSSION TITLE
Weird behaviour using `A-(` and `A-)` with selections on newline characters

### DISCUSSION THREAD

**Discussion by emilyyyylime** [helix-editor/helix#10170] [General]
### Summary

With selection(s) on newline characters using `A-)` and `A-(` sometimes moves selections around weirdly, not just moving the newlines around. I basically understand why this is happening but this is less than ideal behaviour

### Reproduction Steps

[![asciicast](https://asciinema.org/a/oLxpQgL3w7MDtJhg2pYWx4H6R.svg)](https://asciinema.org/a/oLxpQgL3w7MDtJhg2pYWx4H6R)

### Helix Version

helix 22.08.1 (5b73c8c7)

**Comment by archseer** [comment]
It's expected behaviour, since you're selecting the empty line. You can use `_` to trim away whitespace, then `A-)` will behave as you want.

**Comment by emilyyyylime** [comment]
I get that but wouldn't it make more sense for it to not work like that? The bottom cursor moving two lines down makes little sense to me

**Comment by archseer** [comment]
You have three selections total, the last one spanning two lines. `A-(` just rotates selections.
